x/gamm/keeper/multihop.go

```diff
@@ -58,6 +63,12 @@ func (k Keeper) MultihopSwapExactAmountOut(
 
 	insExpected[0] = tokenInMaxAmount
 
+	swapFeeMultiplier := sdk.OneDec()
+
+	if types.SwapAmountOutRoutes(routes).IsOsmoRoutedMultihop() {
+		swapFeeMultiplier = sdk.NewDecWithPrec(5, 1)
```

Comment:
should it not be (5,2)?

Reply:
(5,2) means 0.05
if we want to half fee, we need to use 0.5 multiplier